app/src/main/AndroidManifest.xml

```diff
@@ -15,6 +15,7 @@
         android:theme="@style/AppTheme">
         <activity
             android:name=".PdfViewer"
+            android:launchMode="singleTop"
```

Comment:
I think declaring `android:documentLaunchMode="always"` instead of changing `launchMode` would be better.
https://developer.android.com/guide/topics/manifest/activity-element#dlmode

Then, OS will create a new activity each time PDF is opened, and will show each PDF as a separate item in the task switcher UI

Reply:
It can create to many task instance in task switcher UI / recent screen. tho unsure if that's what we want.

Reply:
I think it's probably what we want.

Reply:
Document mode is probably what we want.

Reply:
okay, thanks.